scripts/deploy_app.sh

```diff
@@ -34,11 +34,10 @@ main() {
   export TEMP_FILE="${APP}_deploy.yaml"
   < "$APP" \
     sed -E "s/SERVICE_NAME/${service_name}/g" \
-    | sed -E "s/SERVICE_CONFIG_ID/${config_id}/g" \
     > "$TEMP_FILE"
   echo "Deploying ${APP}..."
   echo "gcloud -q app deploy $APP"
```

Comment:
fix this echo statement, too. also, why does it say $APP and not $TEMP_FILE, is that intentional/ok?

Reply:
I think it's currently correct. It says "deploying [app name]"
 instead of "deploying [app file]". At least it looks like this was original intention.